hipercow-api-unit-tests/stylecop.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  // ACTION REQUIRED: This file was automatically added to your project, but it
+  // will not take effect until additional steps are taken to enable it. See the
+  // following page for additional information:
+  //
+  // https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/EnableConfiguration.md
```

Comment:
Has this dire warning been followed? Do you need to add stylecopy to your gh action or does it get triggered elsewhere?

Reply:
Ah - yes, this has been followed (without it, I think stylecop doesn't work in some way! - But I haven't removed it from the template file. I'll zap it (and also in the main project) in #3